README.md

```diff
@@ -1,2 +1,13 @@
 # eq-lookup-suggestions-data
 eQ Lookup Suggestions Data
+
+This repository contains versioned TextField suggestion data for eq-questionnaire-runner
+
+| Datasets |       |
```

Comment:
- Datasets -> Dataset
- The second column should have a heading of "Description"